tests/data/full-transaction-request.json

```diff
@@ -46,6 +46,7 @@
     "decline_code": "invalid number"
   },
   "credit_card": {
+    "country": "US",
```

Comment:
How about another request that contains country but no issuer ID? I think we need to verify that no exception is thrown in that case.

Reply:
We don't throw an exception if they provide both. Also, the change to `test_validation.py` would test that no exception is thrown when they only provide a valid `country`.